vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -12,8 +12,22 @@ installBcc() {
     dnf_install 120 5 25 bcc-tools || exit $ERR_BCC_INSTALL_TIMEOUT
 }
 
-configGPUDrivers() {
-    echo "Not installing GPU drivers on Mariner"
+addMarinerNvidiaRepo() {
+    if [[ $OS_VERSION == "2.0" ]]; then 
```

Comment:
nit: a bit funny to do this here vs at the function call site in install-deps.sh